core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/visualization/barChart/BarChartOpExec.java

```diff
@@ -13,25 +14,24 @@
 public class BarChartOpExec extends MapOpExec {
 
     private final BarChartOpDesc opDesc;
+    private final OperatorSchemaInfo operatorSchemaInfo;
 
-    public BarChartOpExec(BarChartOpDesc opDesc) {
+    public BarChartOpExec(BarChartOpDesc opDesc, OperatorSchemaInfo operatorSchemaInfo) {
         this.opDesc = opDesc;
+        this.operatorSchemaInfo = operatorSchemaInfo;
         this.setMapFunc((Function1<Tuple, Tuple> & Serializable) this::processTuple);
     }
 
     public Tuple processTuple(Tuple t) {
-        List<Object> resultObjects = new ArrayList<>();
-        List<Attribute> resultAttributes = new ArrayList<>();
-
+        Tuple.BuilderV2 builder = Tuple.newBuilder(operatorSchemaInfo.outputSchema());
```

Comment:
This looks ok.. Can you please run one of these visualizations to see that everything runs ok?

Reply:
I've been trying to run the viz operators, and I'm not able to test them because they seem to not be functional. Attached is a screenshot of the UI code till where I see a proper response of the data that needs to be visualised. 
![data-for-viz-op](https://user-images.githubusercontent.com/6594255/123558603-036ea000-d74c-11eb-9ff6-e10b08d7efd0.png)

PS: In `visualization-panel-content.component.ts`, there is a member called `this.columns` which is never initialised, but used to populate data for line/pie/bar charts. I think that's the reason why no data is showing up.